news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "\"Medal Hunter\" Title's Second Coming"
+date: 2023-08-?? 03:00:00 +0000

```

Comment:
```suggestion
date: 2023-08-28 03:00:00 +0000
```

Reply:
unsure if this somehow flags on remark but better to have a placeholder like this instead of question marks